src/test/java/de/qabel/desktop/ui/LayoutControllerTest.java

```diff
@@ -31,14 +31,14 @@ public void testFillQuotaInformation() {
         LayoutController controller = createController();
         assertEquals("30%", controller.quota.getText());
         assertEquals(30, (int) controller.quotaBar.getMinWidth());
-        assertThat(controller.quotaDescription.getText(), containsString("667.57 MB"));
+        assertThat(controller.quotaDescription.getText(), containsString("286.1 MB"));
```

Comment:
There you go. These ugly numbers are just bad to read and debug. Who shall know if that's correct ... Please choose numbers that are easy to reason about, because the test should be easy to read and errors should be easy to debug.
Given I execute the tests and get "assertion failed: expected <286.1 MB> but got <667.67 MB>". Because we know the answer it might be  a little bit more obvious that it's ~1/3 of something compared to ~2/3 of something... but it would be far easier to understand it if we hat clean numbers: "assertion failed: expected <300 MB> but got <700 MB>" or something like that. Especially if you have the definition of the quota inside the test or at least inside the test class:

```JAVA
@Test
public void testFillQuotaInformation() {
    setQuota(1000*1024*1024); // or whatever creates the best numbers for the  test...maybe setQuota(toMb(1000))
    ...
    assertEquals("30%", controller.quota.getText());
    ...
    assertQuotaDescriptionContains("300 MB"));
}
```